packages/RichTextEditing-Core.package/RichTextEditorModel.class/instance/loadFileFrom..st

```diff
@@ -0,0 +1,8 @@
+saving and loading
+loadFileFrom: filePath
+
+	|filestream text document|
+	filestream := StandardFileStream readOnlyFileNamed: filePath.
```

Comment:
Not sure if there are any best practices on that, but I think we should close the `filestream` after reading

Reply:
fixed that

Reply:
Yes definitely. Done in the tests, so I can easily implement this